corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -29,7 +29,7 @@ def interactive_import(self):
         # supress requests library INFO logs
         logging.getLogger("requests").setLevel(logging.WARNING)
 
-        # let user chose a project from the list ['commcare-mobile-2.21', ...]
+        # let user chose a project from the list ['commcare-mobile-2.21.0', ...]
```

Comment:
this is still referring to a jenkins project name so the 'point' should be removed

Reply:
thought that was just an example so didnt revert it.. Fine either ways so reverted now.